### ISSUE TITLE
Make ACLs of events and series editable

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#356]
Users need the ability to change the visibility of series and events, i.e. to give different users or user groups access to events/series. 

This is blocked by two big issues:
- #671 
- #631 

Once those are done, this issue itself probably won't be too hard to implement.

**Comment by dagraf** [comment]
Maybe you allready know this, but: Katrin is working for our own Opencast on integrating the ACL templates used by SWITCH into the core. You could maybe join forces.

**Comment by dagraf** [comment]
In my opinion this feature has as precondition feature https://github.com/elan-ev/tobira/issues/468

> I should be first able to see (in a nice way) what ACL i selected on a video and then I should be able to modify it.

**Comment by oas777** [comment]
We addressed and tabled this in our meeting, but could I have a short update here?

**Comment by JulianKniephoff** [comment]
@oas777 are you talking about the "direct metadata update" stuff? Lars still has to schedule a meeting with at least Matthias and I. DACH probably delayed that.

**Comment by oas777** [comment]
No, I'm interested in the status of the overall feature of editing ACL for existing videos (which we currently only allow when uploading). If I understand Lukas initial comment correctly, this needs
- a UI (done in https://github.com/elan-ev/tobira/issues/671) and
- fast editing of ACL (https://github.com/elan-ev/tobira/issues/631); this would also allow fast metadata editing, which is not my concern here.
- Plus, according to David it needs: https://github.com/elan-ev/tobira/issues/468.

**Comment by owi92** [comment]
The UI from #671 can only be used once #631 is solved, which depends on the outcome of the meeting that Julian mentioned in his comment.

#468 doesn't necessarily depend on #631, so I could theoretically already start working on that.

**Comment by oas777** [comment]
I think #468  is less important than having #671 for existing videos and #631  therefore, but let's wait for @dagraf to speak.

**Comment by dagraf** [comment]
I agree, that #631 is the most important issue since it blocks #671. But as we will definitively need #468 (in my opinion) I would suggest @owi92 should start working on that. Let's discuss this in our todays meeting.

**Comment by LukasKalbertodt** [comment]
From our meeting:

- #468 we can already start (I added some details in that issue)
- Regarding the blocker #631: we decided to try and ignore it. That means Tobira would send changes to Opencast (and potentially start a workflow) and just assume everything was saved.
  - This is in light of discussions around changing Opencast fundamentally, which would also resolve the blocker. That will likely only go in OC 18+ though. 
  -  Tobira would *not* lock an event for the time Opencast is running the workflow on it. That, unlike the other parts of this issue, would be quite a bit of work, touching way too many parts of Tobira. And since in the long term, we will use a proper solution, would be wasted time.
  - This feature would be hidden behind a config option, that explains potential risks with it.
  - We devs have to research how to make it work with the current Opencast APIs.

**Comment by oas777** [comment]
Re-visiting this for Lukas's summary: While "events and series" are in the title, we're talking "events" currently, right?

**Comment by LukasKalbertodt** [comment]
This was already done for events and is now also done for series (in #1311)
